cli/src/apiToken.ts

```diff
@@ -2,16 +2,44 @@ import { getAccessTokenPath } from "./accessTokenPath.ts";
 
 const serverURL = Deno.env.get("RMMBR_SERVER");
 
-export const apiToken = () =>
-  getAccessToken()
-    .then(getOrCreateApiToken)
+type APITokenInterface =
+  | { create: true }
+  | { delete: string }
+  | { list: true }
+  | { get: true }
+  | Record<string, never>;
+
+export const apiToken = (
+  cmd: APITokenInterface,
+) => {
+  const action: (_: string) => Promise<unknown> = "create" in cmd
```

Comment:
it would be shorter and more readable to have a mapping and then work on the mapping

e.g. 

```js
const commandMapping = {delete:  deleteApiToken, ...};
Object.entries.find(...)();
```